src/config.ts

```diff
@@ -21,19 +21,46 @@ import path from 'path';
 
 import { sanitizeForFilePath } from './tools/utils';
 
-import type { Config } from '../config';
-import type { LaunchOptions, BrowserContextOptions } from 'playwright';
+import type { Config, ToolCapability } from '../config';
+import type { LaunchOptions } from 'playwright';
 
-export type BrowserOptions = {
-  browserName: 'chromium' | 'firefox' | 'webkit';
-  launchOptions: LaunchOptions;
-  contextOptions: BrowserContextOptions;
+export type CLIOptions = {
+  browser?: string;
+  caps?: string;
+  cdpEndpoint?: string;
+  executablePath?: string;
+  headless?: boolean;
+  userDataDir?: string;
+  port?: number;
+  host?: string;
+  vision?: boolean;
+  config?: string;
 };
 
-export async function toBrowserOptions(config: Config): Promise<BrowserOptions> {
+const defaultConfig: Config = {
+  browser: {
+    browserName: 'chromium',
+    userDataDir: os.tmpdir(),
+    launchOptions: {
+      channel: 'chrome',
```

Comment:
If the client specifies only browserName, we'll still use 'chrome' instead of 'chromium'.

Reply:
That's as designed, user is in full control.